product-service/handlers/getProduct.ts

```diff
@@ -0,0 +1,31 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import productList from '../productList.json';
+
+export const getProduct: APIGatewayProxyHandler = async (event, _context) => {
+	const { productId } = event.pathParameters;
```

Comment:
`pathParameters` can be null.
You can do something like this:
```js
const productId = event.pathParameters?.productId;
```

Reply:
Fixed